### PR TITLE
[ticket/13875] Ignore cache, ext, and store folder in lint test

### PR DESCRIPTION
**Pull request by marc1706**
Ticket: https://tracker.phpbb.com/browse/PHPBB3-13875

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -22,6 +22,9 @@
         </testsuite>
         <testsuite name="phpBB Lint Test">
             <file>./tests/lint_test.php</file>
+            <exclude>./phpBB/cache</exclude>
```
Comment (nickvergessen) [inline phpunit.xml.dist]: https://github.com/phpbb/phpbb/blob/3.1.x/tests/lint_test.php#L82-L89

**Comment by Nicofuma** [comment]
And for the vendors? What do we want to do?

**Comment by marc1706** [comment]
@Nicofuma vendors are already excluded :palm_tree:

**Comment by bantu** [comment]
Will extension authors complain about the ext directory no longer being covered?

**Comment by nickvergessen** [comment]
I think they should use a separate one

**Comment by bantu** [comment]
Aye!

**Pull request merged**